charts/ibm-watson-tts-embed-runtime/values.yaml

```diff
@@ -96,6 +96,12 @@ models:
     digest: sha256:3595ecc0561f0228d4141edfdd62febdee8a68a7b62b4dad5d055da7f59f95d2
     tag: 1.0.0
     enabled: false
+  enUSHenryV3Voice:
```

Comment:
Thanks for double-checking if we missed any others!
Yeah, I switched back to "models" instead of "voices" for consistency. The generic term "models" should work across all of the embed products. We lose some specificity for consistency/generality. What are your thoughts on it, do you think it will be common source of confusion?

Reply:
I feel like as long as we document usage, I like having models for both. Just wanted to check!